plan/plan.go

```diff
@@ -539,9 +586,13 @@ func resolveComponentCommon(commons ...v2.Common) ComponentCommon {
 	if oktaConfig != nil {
 		oktaPlan = &OktaProvider{
 			OrgName: *oktaConfig.OrgName,
-			Version: oktaConfig.Version,
 			BaseURL: oktaConfig.BaseURL,
 		}
+
+		providerVersions["okta"] = ProviderVersion{
+			Source:  "oktadeveloper/okta",
```

Comment:
We're currently running on a[ forked version](https://github.com/chanzuckerberg/terraform-provider-okta) of this provider

Reply:
erg, is that in the registry?

Reply:
not yet, but I can figure that out

Reply:
I can handle it - this is blocking my work and I think I know how to get it published quickly.

Reply:
ok, thank you!

Reply:
https://registry.terraform.io/providers/chanzuckerberg/okta/latest